client/post-editor/media-modal/edit/_style.scss

```diff
@@ -0,0 +1,47 @@
+.editor-media-modal-edit__content {
+	background: black;
```

Comment:
We avoid using custom colors. If you take a look at _colors.scss, this is what our styleguide currently supports. Maybe try `$gray-dark` here? 

cc @drw158 @adambbecker